donkeycar/tests/test_train.py

```diff
@@ -13,7 +13,7 @@
 from donkeycar.utils import get_model_by_type, normalize_image, train_test_split
 
 Data = namedtuple('Data',
-                  ['type', 'name', 'convergence', 'pretrained', 'preprocess'],
+                  ['type', 'name', 'convergence', 'pretrained', 'preprocess', 'tf_lite', 'tensor_rt'],
                   defaults=(None, ) * 5)
```

Comment:
If you add 2 more arguments to the namedtuple, please also increase the default initialisation by 2, i.e. 5 -> 7.

Reply:
This is now resolved